packages/console/src/components/PlanUsage/PlanUsageCard/index.tsx

```diff
@@ -78,29 +126,86 @@ function PlanUsageCard({
           <Trans
             components={{
               span: (
-                <span className={classNames(styles.usageTip, isUsageTipHidden && styles.hidden)} />
+                <span
+                  className={classNames(
+                    styles.usageTip,
+                    // Hide usage tip for free plan users.
+                    (!isPaidTenant || basicQuota === undefined) && styles.hidden
+                  )}
+                />
               ),
             }}
           >
-            {t(usageKey, {
-              usage:
-                quota === undefined
-                  ? formatNumber(usage)
-                  : typeof quota === 'number'
-                  ? `${formatNumber(usage)} / ${formatNumber(quota)}${
-                      usagePercent === undefined ? '' : ` (${(usagePercent * 100).toFixed(0)}%)`
-                    }`
-                  : `${formatNumber(usage)} / ${String(t('subscription.quota_table.unlimited'))}`,
-            })}
+            {t(
+              (() => {
+                switch (true) {
+                  case basicQuota === null || basicQuota === true: {
+                    return 'subscription.usage.usage_description_with_unlimited_quota';
+                  }
+                  case basicQuota === false || basicQuota === 0: {
+                    return 'subscription.usage.usage_description_without_quota';
+                  }
+                  case typeof basicQuota === 'number': {
+                    return 'subscription.usage.usage_description_with_limited_quota';
+                  }
+                  default: {
+                    return usageKey;
+                  }
+                }
+              })(),
```

Comment:
Maybe we can use a component to replace this logic.

Reply:
Per discussion, we use early returned if statements instead since it's clear enough.